gulpfile.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var gulp = require('gulp');
+var jeditor = require('gulp-json-editor');
+var merge = require('merge-stream');
+var zip = require('gulp-zip');
+
+var files = ['background.js', 'contentScript.js', 'logo.png',
```

Comment:
I'd be okay adding a `src` directory! I don't think zipping the current directory or zipping the `src` directory (to install the extension locally) is any different, so this is totally fine with me! Let me re-jiggle the master branch and see if it looks good with you :)